client/options/nowPlaying.ts

```diff
@@ -57,6 +57,27 @@ const radios: RadioRecord[] = [
 			return { np, listeners, dj } as RadioData;
 		},
 	} as RadioRecord,
+	{
+		urlBase: 'https://radio.shamik.ooo/',
```

Comment:
I'd like to avoid hardcoding the domain as much as possible. Please make this a server config.